README.md

```diff
@@ -27,7 +27,7 @@ The scripts have been implemented to be run inside the Docker container only. Th
 - Build the `Docker` image:
 
 ```bash
-docker build . -t topo-imagery
+docker build --build-arg=GIT_HASH=dev --build-arg=GIT_VERSION=dev --tag=topo-imagery .
```

Comment:
do we need to make the local build commands more complicated? what value does adding "dev"/"dev" add here?

Reply:
There was some discussion about the best way to check that these exist when running in CI, open to suggestions! I think you are right we should change it as I can foresee they will end up commented out in the Dockerfile which would be a bad sign.

Reply:
This is the first command in the README.md it is likely what people will run to first build this system, is making this command much more complicated to add "dev" "dev" seem excessive?

Reply:
The reason for making these mandatory was to ensure that we don't forget to add them, which would only be detected at runtime.

Reply:
Changed to make these optional.